### ISSUE TITLE
Ashen band of endless destruction Missing in game for me. 

### ISSUE THREAD

**Issue by mkubiak402** [dalaranwow/dalaran-wow#5694]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
NOTE: I've has a Ticket open in game for months hoping to talk to a GM no one has gotten back to me. 
On Toons "Malinda" and 'Chalee" The vendor of "Ashen band of endless destruction" Anoren and Ormus has a Blue "?" and no rings to give. 

**Expected behaviour**: Tell us what should happen instead

They should have an option to pick what Ring you need
If you lost it an option to just pay gold to get it back

**Steps to reproduce the problem**:

1.  walk in to ICC 
2.  go up to the ring vendor and see a blue ?
3. 

**Include proofs for this behaviour**
here are some pics...

none of them have the ring.. 
https://imgur.com/DmylzSJ
https://imgur.com/4k8EGIY

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

https://www.wowhead.com/item=50398/ashen-band-of-endless-destruction
reputation is Exalted 
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by mkubiak402** [comment]
@mercred so here it is can you help?

**Comment by MalkaviaDW** [comment]
Hello there,

Open a ticket in game and we will help you there. 

Cheers!

**Comment by mkubiak402** [comment]
Yeah i did and it was closed saying i was off line... ?????
